pkg/downloader/manager.go

```diff
@@ -406,7 +406,24 @@ func (m *Manager) getRepoNames(deps []*chartutil.Dependency) (map[string]string,
 		}
 	}
 	if len(missing) > 0 {
-		return nil, fmt.Errorf("no repository definition for %s. Try 'helm repo add'", strings.Join(missing, ", "))
+		if len(missing) > 0 {
+			errorMessage := fmt.Sprintf("no repository definition for %s. Please add them via 'helm repo add'", strings.Join(missing, ", "))
+			// It is common for people to try to enter "stable" as a repository instead of the actual URL.
+			// For this case, let's give them a suggestion.
+			containsNonURL := false
+			for _, repo := range missing {
+				if !strings.Contains(repo, "//") && !strings.HasPrefix(repo, "@") && !strings.HasPrefix(repo, "alias:") {
+					containsNonURL = true
+				}
+			}
+			if containsNonURL {
+				errorMessage += `
+Note that repositories must be URLs. For example, to refer to the stable
```

Comment:
You might want to note here that you can refer to it in the requirements file with an alias (e.g. `@stable`)

Reply:
Done! Sorry, didn't notice this thread.